src/fetch_package.js

```diff
@@ -17,74 +17,30 @@ const vm = require("vm");
 const request = require('superagent');
 const logging = require("winston");
 
-const cache = require("./cache.js");
 const profile = require("./profile.js");
 const graphiteUtil = require("./graphite_util.js");
 
-// fetchPackage takes a cacheBehavior property, which is one of these:
-//    'yes': try to retrieve the object from the cache
-//    'no': do not try to retrieve the object from the cache (but
-//          still store it in the cache after retrieving it).
-//    'ignore': do not try to retrieve the object from the cache,
-//          nor try to store it in the cache.
-//    'ims': retrieve the object from the cache, but use the
-//          last-modified date on the result in order to do an
-//          if-modified-since query.
-// This variable controls the cache behavior that is used if the
-// user does not pass in a value for cacheBehavior for fetchPackage().
-let defaultCacheBehavior;
-
 // How many times we retry on 5xx error or similar, before giving up.
-let numRetries;
+const numRetries = 2;
 
 // What requests are currently in flight?
-let inFlightRequests;
-
-
-const resetGlobals = function() {
-    defaultCacheBehavior = 'yes';
-    numRetries = 2;     // so 3 tries total
-    inFlightRequests = {};
-};
-
-resetGlobals();
-
+const inFlightRequests = {};
 
 /**
  * Given a full url, e.g. http://kastatic.org/javascript/foo-package.js,
  * return a promise holding the package contents.  If requestStats is
- * defined, we update it with how many fetches we had to do and in the case of
- * cacheBehavior==="yes", a `fromCache` count.
+ * defined, we update it with how many fetches we had to do.
  */
-const fetchPackage = function(url, cacheBehavior, requestStats,
+const fetchPackage = function(url, requestStats,
                               triesLeftAfterThisOne) {
```

Comment:
can this now go on the previous line?  (do we run prettier in this repo?)

Reply:
We don't run prettier at the moment (or even eslint, I don't think). It's on my list, but time...